packages/react/README.md

```diff
@@ -51,9 +66,48 @@ import Button from '@massds/mayflower-react/dist/Button';
 ```javascript
 <Button text="Button" onClick={() => console.log('mayflower button clicked!')} />
 ```
+>For a more detailed guide and information on the components included in Mayflower React and their functionality, visit our [Mayflower React Storybook][react-storybook]. Click on the Info and Knobs tabs for component prop types, details and options.
+
+4. Component styles are imported separated, follow the scss modules documentation in each component. Create a scss file and import the necessary styles from mayflower-assets for the React component in use. E.g. If you using the slim header and slim footer components, in a SCSS files, import these styles:
+```scss
+// Header
+@use "~@massds/mayflower-assets/scss/01-atoms/button-with-icon";
+@use "~@massds/mayflower-assets/scss/02-molecules/brand-banner";
+@use "~@massds/mayflower-assets/scss/03-organisms/header-slim";
+// Footer
+@use "~@massds/mayflower-assets/scss/01-atoms/image";
+@use "~@massds/mayflower-assets/scss/01-atoms/site-logo";
+@use "~@massds/mayflower-assets/scss/03-organisms/footer-slim";
+```
+
+5. Config SCSS import paths
+When using this mayflower-assets for the .scss files, you will need to include its include paths, see [mayflower-assets](packages/assets/README.md) for more info.
+
+
+### To consume the typed components in your typescript project
+1. Create a `mayflower.d.ts` in your project and declare module imports by mayflower-react:
```

Comment:
This is actually an optional step. It's only necessary if the project is directly importing something from `@massds/mayflower-assets` into a TS code. It turned out we could even remove it from Data Hub, but let's keep it for now.